src/app.module.ts

```diff
@@ -21,16 +22,21 @@ import {PrismaModule} from './prisma/prisma.module';
         transport:
           process.env.NODE_ENV !== 'production'
             ? {
-              target: 'pino-pretty',
-              options: {
-                singleLine: true,
-                colorize: true,
-              },
-            }
+                target: 'pino-pretty',
+                options: {
+                  singleLine: true,
+                  colorize: true,
+                },
+              }
             : undefined,
       },
     }),
-
+    ThrottlerModule.forRoot([
+      {
+        ttl: 60000, // The time to keep records of requests (in seconds)
+        limit: 10, // The maximum number of requests allowed in the time window
```

Comment:
I think we should probably move this to a config file with proper comments about units being used